### ISSUE TITLE
Failed to start a new session, ERROR: Failed to ensure session integrity! Aborting session initialization...

### ISSUE THREAD

**Issue by Jassaboi** [SafeExamBrowser/seb-win-refactoring#992]
Log Files:
[2024-10-10_10h19m35s_Runtime.log](https://github.com/user-attachments/files/17324989/2024-10-10_10h19m35s_Runtime.log)
This is the same every time I startup the application;
[2024-10-07_14h14m02s_Runtime.log](https://github.com/user-attachments/files/17324992/2024-10-07_14h14m02s_Runtime.log)

BUG:
If starting up SEB, it gives an error message shown in the attachedscreenshot 

1. Start-up SEB
2. Wait
3. See error

Expected behaviour:
-That it would startup

Screenshots:
![Screenshot 2024-10-10 101947](https://github.com/user-attachments/assets/69ab78f6-1207-4027-ab57-8018a9ace9ed)
(Idk if this worked, I'm not that familiar with github)

Version information
 - OS Build 22621.4249, version 22H2 (windows 11)
 - SEB-Version 3.8.0

Additional context
-Other students from my group didn't have this problem.
-I've already uninstalled and reïnstalled the application and already restarted my laptop

**Comment by strau0106** [comment]
You have Custom cursors  enabled. You can not use custom cursors, you could be cheating with those.

**Comment by dbuechel** [comment]
That is exactly right and is furthermore stated in the runtime log in plain English:

```
2024-10-07 14:14:17.628 [09] - INFO: [SystemSentinel] Starting cursor verification...
2024-10-07 14:14:17.633 [09] - WARNING: [SystemSentinel] Configuration of cursor 'AppStarting' is compromised: 'C:\Users\User\Documents\Cursors\Groen-Laad-ding.cur'!
2024-10-07 14:14:17.634 [09] - WARNING: [SystemSentinel] Configuration of cursor 'Arrow' is compromised: 'C:\Users\User\Documents\Cursors\Groen-cartoon.cur'!
2024-10-07 14:14:17.635 [09] - WARNING: [SystemSentinel] Configuration of cursor 'Hand' is compromised: 'C:\Users\User\Documents\Cursors\Groen-cartoon.cur'!
2024-10-07 14:14:17.636 [09] - WARNING: [SystemSentinel] Configuration of cursor 'Wait' is compromised: 'C:\Users\User\Documents\Cursors\Groen-Laad-ding.cur'!
2024-10-07 14:14:17.637 [09] - WARNING: [SystemSentinel] Cursor configuration is compromised!
```
